ledger-tool/src/main.rs

```diff
@@ -2065,6 +2065,9 @@ fn main() {
                     accounts_index_config.index_limit_mb = Some(limit);
                 } else if arg_matches.is_present("disable_accounts_disk_index") {
                     accounts_index_config.index_limit_mb = None;
+                } else {
+                    // disk acct idx on by default
+                    accounts_index_config.index_limit_mb = Some(10_000);
```

Comment:
Is there a good place that we can name `10_000` something like `DEFAULT_ACCOUNTS_INDEX_LIMIT_MB` ?